org.compiere.db.oracle.provider/src/org/adempiere/db/oracle/config/ConfigOracle.java

```diff
@@ -357,9 +357,10 @@ public String test(IDBConfigMonitor monitor, ConfigurationData data)
 		}
 		//
 		//	URL (derived)	jdbc:oracle:thin:@//prod1:1521/prod1
-		String url = "jdbc:oracle:thin:@//" + databaseServer.getHostName()
-			+ ":" + databasePort
-			+ "/" + databaseName;
+		String url = "jdbc:oracle:thin:@"
+			+ (server != null && server.length() > 0
+				? "//" + databaseServer.getHostName() + ":" + databasePort + "/" + databaseName
+				: databaseName);
```

Comment:
this wouldn't works with console-setup-alt.sh since in console mode, you can't enter empty string for database server host name.